src/components/common/Navigation.js

```diff
@@ -4,17 +4,36 @@ import { Menu } from 'antd';
 import { Link } from 'react-router-dom';
 
 function Navigation() {
-    const [current, setCurrent] = useState(null);
-    
-    return(
-        <Menu onClick={({key}) => setCurrent(key)} selectedKeys={[current]} mode="horizontal">
-            <Menu.Item key="home"><Link to ="/"/>Home</Menu.Item>
-            <Menu.Item key="case-view"><Link to="/case-view"/>Cases</Menu.Item>
-            <Menu.Item key="calandar"><Link to="/calendar" />Calendar</Menu.Item>
-            <Menu.Item key="resources"><Link to="/resources" />Staff Resources</Menu.Item>
-            <Menu.Item key="services"><Link to="/services" />Services</Menu.Item>
-        </Menu>
-    );
-};
+  const [current, setCurrent] = useState(null);
 
-export default Navigation;
\ No newline at end of file
+  return (
```

Comment:
were the changes to this block of code just the prettier formatter?

Reply:
Yeah, if this causing conflicts in the future, we may need to add a `.prettierrc` file to standardize our formatting.